File: ahma0/PGS_12914.kt
class Solution {
    
    fun solution(n: Int): Long {
        var answer: Long = 0
        
        val arr = LongArray(n + 1, {0})
        arr[0] = 1
        arr[1] = 1
        
        for(i: Int in 2 until n + 1) {
            arr[i] = (arr[i - 1] + arr[i - 2]) % 1234567
        }
        
        return arr[n]
    }

}

File: chaeryeon823/PGS_12914.js
// 멀리 뛰기
// lv 2
// 피보나치 수, 구현

function solution(n) {
  var answer = 0;
  if (n === 1) {
    return 1;
  }
  [a, b] = [1, 1];
  for (let i = 1; i < n; i++) {
    [a, b] = [b, (a + b) % 1234567];
  }

  return b;
}
